evergreen/config.yml

```diff
@@ -906,7 +913,7 @@ tasks:
       test_filter: CoreTests
 
 - name: core-tests
-  tags: [ "test_suite", "for_pull_requests" ]
+  tags: [ "test_suite", "for_pull_requests", "local_tests" ]
```

Comment:
What would you think about updating `"test_suite"` to include the `object-store-tests` instead of creating a third tag? That test typically completes in less than 5 minutes

Reply:
Sure.